src/custom-elements/src/components/spinner/spinner.tsx

```diff
@@ -0,0 +1,51 @@
+import { Component, Prop, Watch } from '@stencil/core';
+import { VALID_COLORS } from '../../utils/color';
+import { VALID_SIZES } from '../../utils/size';
```

Comment:
I like the idea of VALID_COLORS and VALID_SIZES, but they are not shared across all components. 
The sizes the spinner use are the same as the sizes used in icons (xs, sm, sm--2, sm--3, md, lg, xl, xxl), but not the same sizes used in buttons (sm, md, lg and xl) or others. So, please update the names accordingly.  
VALID_ICON_SIZES and VALID_TEXT_COLORS. 
Valid text colors are defined in https://assets.ctl.io/chi/1.1.1/utilities/text/
It might be a good idea to share all this variables in a JSON file for the rest of Chi stuff (pug documentation, sass files, etc), but for the moment your solution is enough. TY.

Reply:
sure thing thanks.

Reply:
Please, @srinath-koneru , commit the requested changes to continue with the PR lifecycle. TY.